plugins/rollbar_dashboards/lib/rollbar_dashboards/client.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module RollbarDashboards
+  class Client
+    def initialize(dashboard)
+      @base = dashboard.base_url
+      @token = dashboard.resolved_read_token
+    end
+
+    def top_errors(hours: 24, environments: ['production'])
+      args = {
+        access_token: @token,
+        hours: hours,
+        environments: environments.join(',')
+      }.to_query
+
+      response = Faraday.get("#{@base}/reports/top_active_items?#{args}")
+      if response.success?
+        data = ::JSON.parse(response.body)
+        data['result'].map { |item_hash| item_hash['item'] }
+      end
+    rescue ::JSON::ParserError
+      nil
+    end
+
+    def items(additional_params = {})
+      params = {
```

Comment:
since additional_params is not reused I would normally do something like:

```
params = {foo: bar}.merge(params)
````

Reply:
also don't name different data-structures the same .. params -> hash ... query -> string